src/NRedisStack/Json/JsonCommands.cs

```diff
@@ -60,6 +60,37 @@ public bool Set(RedisKey key, RedisValue path, RedisValue json, When when = When
         return true;
     }
 
+    /// <inheritdoc/>
+    public bool SetFromFile(RedisKey key, RedisValue path, string filePath, When when = When.Always)
+    {
+        string fileContent  = File.ReadAllText(filePath);
+        // TODO: Check if the content is valid JSON
+        return Set(key, path, fileContent, when);
+    }
+
+    /// <inheritdoc/>
+    public int SetFromDirectory(RedisValue path, string filesPath, When when = When.Always)
+    {
+        int inserted = 0;
```

Comment:
This one looks too specific to me.. I don't think we should implement this function

BTW, I think that the users will want to warp all the `SetFromFile` calls with a MULTI

Reply:
I can agree on a multi. We have this in python - hence wanted to get it into other clients.

Reply:
I will implement support for Transactions and pipelines in the future and then I will change this command